runtime-manager/modules/ROOT/pages/deployment-strategies.adoc

```diff
@@ -35,12 +39,13 @@ Icons appear at the top of each page in the Runtime Manager documentation, indic
 
 |===
 |Deployment|Content Applies |Content Doesn't Apply
-|CloudHub
-|image:logo-cloud-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
-|image:logo-cloud-disabled.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub"]
-
-
+|CloudHub 2.0
+|image:logo-cloud2-active.png[xref="deployment-strategies.adoc#cloudhub-deployments", title="CloudHub 2.0"]
```

Comment:
cloudhub2-deployments on lines 43 and 44?

Reply:
This reminds me that, after you figure out the HTML with Taylor, you'll want to update **ALL** the RTM topics with the new icons and links.

Reply:
Yes. We will meet with a CH2 dev us to help us confirm what topics should have the blue and which ones should have the grey icon.